src/net/if_.rs

```diff
@@ -17,3 +17,32 @@ pub fn if_nametoindex<P: ?Sized + NixPath>(name: &P) -> Result<c_uint> {
         Ok(if_index)
     }
 }
+
+libc_bitflags!(
+    /// Standard interface flags, used by `getifaddrs`
+    pub struct InterfaceFlags: libc::c_int {
+        IFF_UP;
+        IFF_BROADCAST;
+        IFF_DEBUG;
+        IFF_LOOPBACK;
+        IFF_POINTOPOINT;
+        #[cfg(not(any(target_os = "freebsd", target_os = "dragonfly", target_os = "haiku")))]
```

Comment:
Please alphabetize all OSes listed in `#[cfg`s. Additionally please use a whitelist instead of a blacklist for targets for consistency.